onnxruntime/core/session/provider_bridge_ort.cc

```diff
@@ -1138,6 +1138,9 @@ OrtCUDAProviderOptionsV2 OrtCUDAProviderOptionsToOrtCUDAProviderOptionsV2(const
   cuda_options_converted.default_memory_arena_cfg = legacy_cuda_options->default_memory_arena_cfg;
   // Use default value as this field is not available in OrtCUDAProviderOptions
   cuda_options_converted.cudnn_conv_use_max_workspace = 0;
+  cuda_options_converted.alloc = NULL;
```

Comment:
use `nullptr` if possible.

Reply:
This field is removed. We don't expose `alloc`, `free`, and `empty` in C struct API. Instead, we use `ProviderOptions` to create EPs for LazyTensor.